app/models/user.rb

```diff
@@ -2,8 +2,9 @@
 # Fields are all devise settings; most of the methods relate to call list mgmt.
 class User
   include Mongoid::Document
-  include Mongoid::Enum
   include Mongoid::Userstamp::User
+  extend Enumerize
+  extend ActiveModel::Naming
```

Comment:
I think I just copied it there without thinking too hard about it. I removed it, ran a test, and it made no difference. Good catch!